### PR TITLE
Remove duplicated modules from the list on install

### LINKED ISSUE #2105: Table already exists error fails query installing multiple modules and one is already exists
The query fails, but it should probably skip it and carry on with the rest of the modules to install

### PR DESCRIPTION
**Pull request by aguillenv**
Resolves #2105 

Before:
image

After:
image

### CODE REVIEW

**Review by dfellis — APPROVED** [review]

**Comment by dfellis** [comment]
The new test is failing here: https://github.com/iasql/iasql/actions/runs/4377646129/jobs/7661411984#step:6:6743

File: test/basic_integration/install.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  it('reinstalls existing modules', install(['aws_ec2', 'aws_vpc']));
+
+  it('uninstalls all', uninstallAll());
+
+  it('installs complex and dependent sql module', install(['aws_ecs_fargate', 'aws_ecs_simplified']));
+
+  it('fails having an install with a typo', done =>
+    void query(`
+    SELECT * FROM iasql_install(['aws_ec3'])
```
Comment (dfellis) [inline test/basic_integration/install.ts:66]:
```suggestion
    SELECT * FROM iasql_install('aws_ec3')
```

**Review by dfellis — APPROVED** [review]

File: test/basic_integration/install.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    ),
+  );
+
+  it('installs a single module', install(['aws_lambda']));
+
+  it('installs another module that share dependencies with the previous one', install(['aws_ec2']));
+
+  it('uninstalls all', uninstallAll());
+
+  it('installs having a duplication in the list', install(['aws_ecr', 'aws_ecr']));
```
Comment (depombo) [inline test/basic_integration/install.ts:54]: would uninstalling duplicates blow up?

Comment (mtp1376) [reply test/basic_integration/install.ts:54]: Same question as @depombo

**Review by mtp1376 — COMMENTED** [review]
It's :stamp: after the test with duplicate module name on `iasql_uninstall` is added.

File: test/basic_integration/install.ts (hunk 2 above)
Comment (aguillenv) [reply test/basic_integration/install.ts:63]: good catch!

**Review by dfellis — APPROVED** [review]

**Review by depombo — APPROVED** [review]

**Pull request merged**